source/Playnite/Localization.cs

```diff
@@ -46,6 +46,16 @@ public static string CurrentLanguage
             get; private set;
         } = SourceLanguageId;
 
+        public static CultureInfo CultureInfo

```

Comment:
I don't see CultureInfo and IsRightToLeft used anywhere. What are these for?

Reply:
At first I thought of storing the CultureInfo only in Localization instead of the thread. But if you think it would be better not to set the thread culture, then it's back to these properties to be queried instead.